doc/guide/Guide_Introduction.ipynb

```diff
@@ -12,13 +12,17 @@
     "According to the IPCC [[1]](#1), natural risks emerge through the\n",
     "interplay of climate and weather-related hazards, the exposure of goods\n",
     "or people to this hazard, and the specific vulnerability of exposed\n",
-    "people, infrastructure and environment. The unit chosen to measure risk\n",
-    "has to be the most relevant one in a specific decision problem, not\n",
-    "necessarily monetary units. Wildfire hazard might be measured by burned\n",
-    "area, exposure by population or replacement value of homes and hence\n",
-    "risk might be expressed as number of affected people in the context of\n",
-    "evacuation, or repair cost of buildings in the context of property\n",
-    "insurance.\n",
+    "people, infrastructure and environment. In CLIMADA, vulnerability is\n", 
```

Comment:
I would leave out this sentence here as it is a bit of a repetition of what comes below: "In CLIMADA, vulnerability is operationalized through the use of impact functions, which are essential in determining the extent to which exposure is affected by a given hazard."